docs/specs/git.yml

```diff
@@ -754,3 +754,107 @@ outputs:
       "original_content_git_url": "https://github.com/user-assigned/original/blob/original/index.md",
       "gitcommit": "https://github.com/testowner/testrepo/blob/450790f8a588ed8d96f4cb931c744d3855b7a809/index.md",
     }
+---
+# Put contributor list to .publish.json (Expose Contributor Information)
```

Comment:
This file contains a extensive suite of test cases for `_op_gitContributorInformation`. For this particular change, instead of adding new test cases, prefer add check to the new metadata in addition to `_op_gitContributorInformation` check.

It is optional to verify `.publish.json` since we have a orthogonal test case to ensure `.publish.json` contains all system metadata. Technically this change is about adding a new metadata, not adding a property to `.publish.json`.

Reply:
Will create a new pr to do this.